docs/tutorials/create-a-resource-node.md

```diff
@@ -32,7 +32,7 @@ You are presented with a form showing a large number of metadata fields. Fill ou
 <!-- We should really include creating/adding a name here or in another tutorial!! It's a source of immediate confusion and frustration! -->
 
 
-Near the bottom of the form, the _System_ section contains mandatory fields that configure Islandora behaviours. Select **Image** from the _Model_ drop down box. This will enable image display behaviours, and cause image derivatives to be generated once we upload a file.
+Near the bottom of the form, the _System_ section contains mandatory fields that configure Islandora behaviours. Select **Image** from the _Model_ drop box. This will enable image display behaviours, and cause image derivatives to be generated once we upload a file.
```

Comment:
I'm also not sure about this one, "drop down box" is the term for the web form element. This is definitely lingo, but I'm not sure that "drop box" is more clear?